Analysis/include/EventInfo.h

```diff
@@ -99,6 +99,13 @@ class EventInfo {
     bool PassNormalTriggers();
     bool PassVbfTriggers();
 
+    template <class Vector1, class Vector2>
+    inline bool FindGenMatch(const Vector1& v1, const Vector2& v2, double deltaR)
```

Comment:
should be
`boost::optional<size_t> FindGenMatch(const JetCandidate& jet) const`